Improve caps (#463)
membraneframework/membrane_core: CHANGELOG.md

 * Log info about element's playback state change as debug, not as debug_verbose. [#430](https://github.com/membraneframework/membrane_core/pull/430)
 * Rename `Membrane.Time.to_<unit name>/1` into `Membrane.Time.round_to_<unit name>/1` to indicate that the result will be rounded. Make `Membrane.Time.<plural unit name>/1` accept `%Ratio{}` as an argument. Add `Membrane.Time.round_to_timebase/2` function.
 * New `spec` action syntax - the structure of pipeline is now defined with the use of `Membrane.ChildrenSpec` struct.

>  * Pass Elixir patterns to `:caps` in pad definition.

Reviewer: ```suggestion
 * Use Elixir patterns as `:caps` in pad definition.
```
